src/src/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -1255,6 +1255,8 @@ private AuthenticationResult acquireTokenLocalCall(final CallbackHandler callbac
                         ADALError.DEVELOPER_AUTHORITY_IS_NOT_VALID_INSTANCE));
                 return null;
             }
+        } else {
+            Logger.i(TAG , "Authority validation is turned off:" + authorityUrl.getHost(), "");
```

Comment:
or why don't you make authorityUrl.getHost the third parameter

---
In reply to: [66656674](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/568#discussion_r66656674) [](ancestors = 66656674)